src/libYARP_sig/include/yarp/sig/ImageNetworkHeader.h

```diff
@@ -46,6 +46,25 @@ class yarp::sig::ImageNetworkHeader
     yarp::os::NetInt32 paramBlobTag;
     yarp::os::NetInt32 paramBlobLen;
 
+    ImageNetworkHeader()
```

Comment:
@Nicogene Why is it complaining about this? Perhaps the actual issue is in NetInt32? Anyway, please use the initialization list instead or even better (if it fixes the issue) the defaulted constructor (`ImageNetworkHeader() = default;`)

Reply:
Oh wait, this is master... no `default`

Reply:
Clang static analyzer complaints about the missing initialization in the constructor.